config/themes.json

```diff
@@ -158,7 +158,7 @@
                     "HeaderFontFamily": "Consolas, Monaco",
 
                     "CheckBoxBulletDecoratorFontSize": "14",
-                    "CheckBoxMargin": "15,0,0,2",
+                    "CheckBoxMargin": "5,0,0,0.8",
```

Comment:
@MyDrift-user May I ask why in this particular theme the margins are different from the others?
See [the documentation for margins](https://learn.microsoft.com/en-us/dotnet/desktop/wpf/advanced/alignment-margins-and-padding-overview?view=netframeworkdesktop-4.8#understanding-margin-properties) to see how it's structured.

The TL;DR, you can do `Margin="20"` or `Margin="20, 20, 20, 20"`, the second is in clock-wise direction, starting from left (left, top, right, bottom)

Reply:
ups, oversight on my part, will fix it as soon as I get home